### PR TITLE
Docstring cleanup

### PR DESCRIPTION
**Pull request by timhoffm**
## PR Summary

Some docstring cleanup. Mainly capitalization (pydocstyle D403).

There's much more to do, but I don't trust automatic regexp replacements here. Therefore, I'll take it slow and do bits at a time. With that I can do other small doc improvements on the go and the PR does not become too large to review.

### CODE REVIEW

File: lib/matplotlib/rcsetup.py [hunk 1]
```diff
@@ -237,8 +237,8 @@ def validate_int_or_None(s):
 
 def validate_fonttype(s):
     """
-    confirm that this is a Postscript of PDF font type that we know how to
-    convert to
+    Confirm that this is a Postscript of PDF font type that we know how to
```
Comment (anntzer) [inline lib/matplotlib/rcsetup.py]: of -> or

File: lib/matplotlib/rcsetup.py [hunk 2]
```diff
@@ -305,7 +305,7 @@ def __init__(self, n=None, allow_none=False):
         self.allow_none = allow_none
 
     def __call__(self, s):
-        """return a seq of n floats or raise"""
+        """Return a seq of n floats or raise."""
```
Comment (anntzer) [inline lib/matplotlib/rcsetup.py]: a sequence of floats

File: lib/matplotlib/rcsetup.py [hunk 3]
```diff
@@ -329,7 +329,7 @@ def __init__(self, n=None):
         self.n = n
 
     def __call__(self, s):
-        """return a seq of n ints or raise"""
+        """Return a seq of n ints or raise."""
```
Comment (anntzer) [inline lib/matplotlib/rcsetup.py]: `a list of *n* ints

File: lib/matplotlib/rcsetup.py [hunk 4]
```diff
@@ -216,15 +216,15 @@ def validate_dpi(s):
...
+    """Convert s to int or raise."""
     try:
         return int(s)
     except ValueError:
         raise ValueError('Could not convert "%s" to int' % s)
 
 
 def validate_int_or_None(s):
-    """if not None, tries to validate as an int"""
+    """If not None, tries to validate as an int."""
```
Comment (QuLogic) [inline lib/matplotlib/rcsetup.py]: Should rewrite to "Convert ..." like its non-None version.

File: lib/mpl_toolkits/axes_grid1/axes_divider.py [hunk 5]
```diff
@@ -713,7 +713,7 @@ def _calc_offsets(appended_sizes, karray):
 
     def new_locator(self, nx, nx1=None):
         """
-        returns a new locator
+        Returns a new locator
```
Comment (QuLogic) [inline lib/mpl_toolkits/axes_grid1/axes_divider.py]: Create

File: lib/mpl_toolkits/axes_grid1/axes_divider.py [hunk 6]
```diff
@@ -806,7 +806,7 @@ class VBoxDivider(HBoxDivider):
 
     def new_locator(self, ny, ny1=None):
         """
-        returns a new locator
+        Returns a new locator
```
Comment (QuLogic) [inline lib/mpl_toolkits/axes_grid1/axes_divider.py]: Create

File: lib/mpl_toolkits/axes_grid1/parasite_axes.py [hunk 7]
```diff
@@ -251,7 +251,7 @@ def pick(self, mouseevent):
 
     def twinx(self, axes_class=None):
         """
-        create a twin of Axes for generating a plot with a sharex
+        Create a twin of Axes for generating a plot with a sharex
```
Comment (QuLogic) [inline lib/mpl_toolkits/axes_grid1/parasite_axes.py]: s/sharex/shared/

File: lib/mpl_toolkits/axes_grid1/parasite_axes.py [hunk 8]
```diff
@@ -309,7 +309,7 @@ def _remove_twiny(self, ax):
 
     def twin(self, aux_trans=None, axes_class=None):
         """
-        create a twin of Axes for generating a plot with a sharex
+        Create a twin of Axes for generating a plot with a sharex
```
Comment (QuLogic) [inline lib/mpl_toolkits/axes_grid1/parasite_axes.py]: Wait is this even right for `twin`?

**Pull request merged**